mrp_bom_state/views/mrp_bom_view.xml

```diff
@@ -36,6 +36,7 @@
                         <button name="button_draft" type="object" string="Draft" groups="mrp_bom_state.group_mrp_bom_state" attrs="{'invisible':[('state','!=','active')]}" class="oe_highlight" />
                         <button name="button_activate" type="object" string="Activate" attrs="{'invisible':[('state','!=',False)]}" class="oe_highlight" />
                         <button name="button_activate" type="object" string="Activate" attrs="{'invisible':[('state','!=','draft')]}" class="oe_highlight" />
+                        <button name="button_new_version" type="object" string="New Version" attrs="{'invisible':[('state','==','historical')]}" class="oe_highlight" />
```

Comment:
Para que un botón muestre un mensaje de confirmación debéis añadir el atributo 'confirm', creo que ahora mismo cuando pulsáis el botón lo hace sin "avisar" al usuario.

Debéis añadir algo así:
```xml
<button name="button_new_version" type="object"
        string="New Version" attrs="{'invisible':[('state','==','historical')]}"
        class="oe_highlight" confirm="MENSAJE DE CONFIRMACIÓN" />
```